stories/Atom/Buttons-and-labels/Chips/chips_cross.jsx

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import './chips.scss';
+
+export const Chipscross = ({ label }) => {
+  return (
+    <div className="chips">
+      <a class="chips__cross"
+        href="#"
+      >
+        { label }
+      </a>
+    </div>
+  );
+};
```

Comment:
@pallavikadam-srijan can we remove the div and add the class chips on the anchor itself